Cargo.toml

```diff
@@ -33,3 +33,8 @@ cmake = { git = "https://github.com/alexcrichton/cmake-rs" }
 [patch."https://github.com/servo/webrender"]
 webrender = { git = "https://github.com/jdm/webrender", branch = "servo-hl" }
 webrender_api = { git = "https://github.com/jdm/webrender", branch = "servo-hl" }
+
+
+[patch."https://github.com/servo/webxr"]
+webxr = { path = "/home/manishearth/mozilla/Git/webxr/webxr" }
+webxr-api = { path = "/home/manishearth/mozilla/Git/webxr/webxr-api" }
```

Comment:
Must remember to remove these once https://github.com/servo/webxr/pull/12 lands.

Reply:
yep. i'm keeping them in as long as this is a draft PR.